### PR TITLE
Register new File Browser in tools registry

### PR DESCRIPTION
**Pull request by hernanmd**
Register the new file browser in the tools registry after Baseline load.

### CODE REVIEW

File: src/NewTools-FileBrowser/String.extension.st [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Extension { #name : 'String' }
+
+{ #category : '*NewTools-FileBrowser' }
+String >> asFileName [
```
Comment (jecisc) [inline src/NewTools-FileBrowser/String.extension.st]: Is it wanted to have this? I think this kind of methods were removed from Pharo not so long ago?

Comment (hernanmd) [reply src/NewTools-FileBrowser/String.extension.st]: I don't know, but I've checked and two external users use it:

- `Object>>#fuelOut`
- `CodeExporter class>>#writeSourceCodeFrom:baseName:isSt:`

Captura de pantalla 2024-11-07 a las 15 52 47

We can include it for now and open another issue to see if it is needed.

File: src/BaselineOfNewTools/BaselineOfNewTools.class.st [hunk 2]
```diff
@@ -231,7 +231,8 @@ BaselineOfNewTools >> packageRepositoryURLForSpec: spec [
 { #category : 'actions' }
 BaselineOfNewTools >> postload: loader package: packageSpec [
 
-	MCMethodDefinition initializersEnabled: initializersEnabled
+	MCMethodDefinition initializersEnabled: initializersEnabled.
+	Smalltalk tools fileListTool: StFileSystemPresenter.
```
Comment (Ducasse) [inline src/BaselineOfNewTools/BaselineOfNewTools.class.st:235]: please use self class environment no direct reference to Smalltalk!

Comment (jecisc) [reply src/BaselineOfNewTools/BaselineOfNewTools.class.st:235]: The problem is that it is not the environment that holds the tool list so we cannot access it like this

**Comment by jecisc** [comment]
Closing in favor of: https://github.com/pharo-spec/NewTools/pull/885

**Pull request closed without merging**